src/mapnode.h

```diff
@@ -109,6 +109,7 @@ enum Rotation {
 
 
 struct ContentFeatures;
+class Map;
```

Comment:
why ? there is no Map definition outside of this diff, please set this definition in the correct area